failure-accrual/linkerd.yml

```diff
@@ -2,9 +2,17 @@ namers:
 - kind: io.l5d.fs
   rootDir: /disco
 
+telemetry:
+- kind: io.l5d.prometheus
+- kind: io.l5d.recentRequests
+  sampleRate: 0.25
+
+usage:
+  orgId: linkerd-examples-failure-accrual
```

Comment:
Am not sure it matters, but I don't think the usage data collector is going to give us very useful information for this project. This is a test config that's only used for demo purposes, so I wouldn't expect folks to copy it for their own personal use. I think it would be better to just omit the `usage` block here. But `recentRequests` should be fine.